.travis.yml

```diff
@@ -1,4 +1,20 @@
 language: generic
+matrix:
+  include:
+    - name: "generic tests"
+    - name: "flake8 lint on Python 2.7"
+      language: python
+      python: 2.7
+      install: pip install flake8
+      script: flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+    - name: "flake8 lint on Python 3.7"
+      dist: xenial  # required for Python >= 3.7
+      language: python
+      python: 3.7
+      install: pip install flake8
+      script: flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
```

Comment:
Any reason the `install` and `script` lines can't be moved outside the matrix, to the `install` and `script` below?

Reply:
1. We would lose the advantage of parallelism.
2. We would only run tests on legacy Python and not on Python.

Reply:
I don't think that's how it works. Travis CI matrix configuration is often used only to set environment variables (see [strace's](https://github.com/strace/strace/blob/master/.travis.yml) for example), without preventing parallel builds.